docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -706,7 +706,7 @@ Specifies the time (in milliseconds) passed after the `DOMContentLoaded` event,
 
 After the timeout passes or the `window.load` event is raised (whichever happens first), TestCafe starts the test.
 
-> Note that the `DOMContentLoaded` event is raised after the HTML document is loaded and parsed, while `window.load` is raised after all stylesheets, images and subframes are loaded. That is why `window.load` is fired after the `DOMContentLoaded` event with a certain delay.
+> Note that the `DOMContentLoaded` event is raised after the HTML document is loaded and parsed, while `window.load` is raised after all stylesheets, images, and subframes are loaded. That is why there is a delay between the `DOMContentLoaded` and `window.load` events fire.
```

Comment:
- "while `window.load` is raised after all stylesheets"? Why do you use 'while'? Do you mean 'the `DOMContentLoaded` event is raised after the HTML document is loaded and parsed while `window.load` is raised'? But then, how can you add 'after all stylesheets, images, and subframes are loaded'? 
- "That is why there is a delay between the `DOMContentLoaded` and `window.load` events fire" Do you mean 'That is why there is a delay between **when** the `DOMContentLoaded` and `window.load` events fire'? There is also no need to add phrases like 'That is why' - This causes a delay between **when** the `DOMContentLoaded` and `window.load` events fire.

Reply:
This sentence is broken. Does this work:

> The `DOMContentLoaded` event is raised after the HTML document is loaded and parsed. The `window.load` is raised after all stylesheets, images, and subframes are loaded. That causes a delay between the `DOMContentLoaded` and `window.load` events.

OR

> The `DOMContentLoaded` event is raised after the HTML document is loaded and parsed. The `window.load` is raised after all stylesheets, images, and subframes are loaded. That is why there is a delay between the `DOMContentLoaded` and `window.load` events.

Reply:
The DOMContentLoaded event is raised after the HTML document is loaded and parsed. The window.load is raised after all stylesheets, images, and subframes are loaded. This causes a delay between the DOMContentLoaded and window.load events.